editor/test/integration/scope_test.clj

```diff
@@ -55,6 +55,7 @@
               final-node-ids (set (ig/node-ids (g/graph graph-id)))
               new (clojure.set/difference final-node-ids old-node-ids)
               remainders (clojure.set/difference old-node-ids final-node-ids)]
+          (is (= old-count final-count))
```

Comment:
We were already calculating `final-count` but not checking it. I just added a check for it.